src/core/pusher.ts

```diff
@@ -67,6 +68,12 @@ export default class Pusher {
   constructor(app_key : string, options : any) {
     checkAppKey(app_key);
     options = options || {};
+    if (!options.cluster) {
+      let suffix = UrlStore.buildLogSuffix("javascript_quick_start");
+      Logger.warn(
+        `You should always specify a cluster when connecting. ${suffix}`
```

Comment:
However, the [Pusher document](https://pusher.com/docs/clusters) says:

> The default cluster, in us-east-1, has a shortcode of `mt1`. You currently don’t need to specify this as it’s assumed as the default, but there’s no harm in doing so

Actually, though my application uses the default `mt1` cluster, Pusher.js logs a bit strict message "You should always ...".

@hph May I create a pull request to remove this warning? Or, the Pusher document is misdirecting the appropriate usage?

Reply:
Hey @shinnn, the documentation is out of date so I'll update the docs.

Reply:
Long time no see, @hph. 

> the documentation is out of date so I'll update the docs.

The [doc](https://pusher.com/docs/clusters) doesn't seem to be updated and still says:

> You currently don’t need to specify this as it’s assumed as the default, but there’s no harm in doing so.

Can you tell me how will you update it in the future? Will you remove the "`mt1` by default" statement?

Reply:
https://github.com/pusher/pusher-js/pull/261 was merged, that proves my first comment was right.

Reply:
Hi @shinnn! The documentation changes were merged today but they haven't been deployed yet. I was waiting on that before replying to you, but now you know :) We definitely don't want to have an implicit default cluster, so there's no need for a PR to change it, but thank you for offering!